{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/static_source/css/base.scss

```diff
@@ -27,9 +27,8 @@
  *
  */
 
-@import "@/css/components/messages.scss";
-@import "@/css/components/links.scss";
-@import "@/css/components/toasts.scss";
+@import "@/../templates/components/links/links.scss";
```

Comment:
these should be handled by the components and not the base unless they're global

Reply:
links doesn't seem to actually get used anywhere so I'm removing it and messages is linked to the messages.jinja but since it's not in the components directory (it's in templates/util) and doesn't have a linked ts file it wouldn't pick up by the components ts flow